src/components/Menu/Menu/Menu.tsx

```diff
@@ -48,7 +49,7 @@ export interface MenuProps extends VibeComponentProps {
   useDocumentEventListeners?: boolean;
   focusItemIndexOnMount?: number;
   shouldScrollMenu?: boolean;
-  children?: ReactElement | ReactElement[];
+  children?: ReactNode;
```

Comment:
Thanks for contributing! Please use ElementContent, it a type we use for `ReactNode | ReactNode[]`